File: ruby/recursive_count.rb
def recursive_count(num = 0)
  # type your code in here
  puts num if num<10
  recursive_count(num+1) if num<9
end

if __FILE__ == $PROGRAM_NAME
  recursive_count
end

# OPTIONAL
# Please add your pseudocode to this file
# And a written explanation of your solution

File: javascript/recursive_count.js
function recursiveCount(num = 0) {
  // type your code here
  if (num<10) {
    console.log(num);
  }
  if (num<9) {
    recursiveCount(num+1)
  }
}

if (require.main === module) {
  recursiveCount();
}

module.exports = recursiveCount;

// OPTIONAL
// Please add your pseudocode to this file
// And a written explanation of your solution
